ethcore/builtin/src/lib.rs

```diff
@@ -337,6 +366,54 @@ impl Implementation for Sha256 {
 	}
 }
 
+impl Implementation for Blake2F {
+	/// Format of `input`: `[4 bytes for rounds][64 bytes for h][128 bytes for m][8 bytes for t_0][8 bytes for t_1][1 byte for f]`
+	fn execute(&self, input: &[u8], output: &mut BytesRef) -> Result<(), &'static str> {
+		const BLAKE2_F_ARG_LEN: usize = 213;
+		const PROOF: &str = "Checked the length of the input above; qed";
+
+		if input.len() != BLAKE2_F_ARG_LEN {
+			// todo[dvdplm] what's the right 'target' here?
```

Comment:
either `evm` or a new target `builtin` is fine

Reply:
I think `builtin` is fine, can you remove the `todo`?